gpkit/constraints/tight.py

```diff
@@ -3,6 +3,11 @@
 from ..nomials import PosynomialInequality, SignomialInequality
 
 
+class TightnessError(Exception):
+    "Raised when linked variables have differing values."
+    pass
```

Comment:
I think the general MO is to create new types of errors for things that a user might want to catch alone, and feel that TightnessErrors and SubstitutionConflicts are both good examples of that, along with Signomials in GPs and lack of in SPs, etc.